Factory/Barclays_Filter.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Epam.NetMentoring.Factory
+{
+    public class Barclays_Filter : IFilter
```

Comment:
Please use consistent naming across - I see  CamelCase and some `__`- style

Reply:
ok